sw/airborne/modules/computer_vision/cv.h

```diff
@@ -31,10 +31,40 @@
 
 #include "std.h"
 #include "lib/vision/image.h"
+#include "../../peripherals/video_device.h"
```

Comment:
This seems like a circular include. I am not sure where else the peripherals/video_device.h is used but it doesn't seem that useful as it is now as it only includes one thing. One idea is to move the #define VIDEO_FILTER_DEBAYER 0x01 to this file and then only include the computer_vision/cv.h in the peripherals/video_device.h and remove this include here.